assets/src/edit-story/components/library/panes/text/textSets/textSets.js

```diff
@@ -84,25 +90,40 @@ function TextSets() {
       ?.selectedCategory
   );
 
+  const storyPages = useStory(({ state: { pages } }) => pages);
+
   const ref = useRef();
+  const inUseFonts = useRef([]);
```

Comment:
We want to use a ref to store the currently used fonts. If we just make this an on-the-fly computed value, then every single time you move an object or make any change to the editor, it will think things changed. We want to cut down on re-rendering the same filtered text sets.